assets/css/admin.scss

```diff
@@ -235,6 +235,11 @@
 			display: flex;
 			align-items: center;
 			flex-direction: row;
+			justify-content: center;
```

Comment:
I think the `justify-content rules` are backwards, normally would be left but under 500px centered. 🤔

Reply:
I initially wrote it correctly, checked it, then copied the format from a few lines below for consistency and didn't check it again 🤦 updated